docs/adoc/technicalGuideJS/ScoutJS.adoc

```diff
@@ -1288,10 +1288,66 @@ Output:
 
   <scout-version data-value="16.1.0.002"></scout-version>
 
+== TypeScript Support
+
+Eclipse Scout is written in TypeScript and therefore comes with a fully typed TypeScript API.
+
+This gives you many benefits:
+
+* *More efficient development* +
+Types allows your IDE to better assist with code completion during development and only suggests methods and properties for an object that actually exist. +
++
+.Better Code Completion
+image::{imgsdir}/scoutjs_code_completion.png[]
+* *Easier entry* +
+With better code completion and clear method signatures, a not-so-experienced JavaScript developer can more easily get up to speed and understand the relationships in the code.
+* *More security for upgrades* +
+If a used API has changed, the code turns red and the build fails.
+* *More security during development* +
+Incorrect use of an API or code in general is detected at an early stage. The developer gets more security that his code will work. +
```

Comment:
I am not sure if "security" is correct here? E.g. use another word.
E.g. for the last sentence: "The developer gets more confidence that the code will work."
Same for the lines above